README.md

```diff
@@ -100,9 +102,9 @@ Activate-Storage-Access: <token>
 ```
 This is a [structured header](https://datatracker.ietf.org/doc/html/rfc8941) whose value is a [token](https://datatracker.ietf.org/doc/html/rfc8941#section-3.3.4) which is one of the following:
 * `load`: the server requests that the user agent activate the `storage-access` permission before continuing with the load of the resource.
-* `retry`: the server requests that the user agent activate the `storage-access` permission, then retry the request. The retried request must include the `Sec-Fetch-Storage-Access: active` header.
+* `retry`: the server requests that the user agent activate the `storage-access` permission, then retry the request. The retried request must include the `Sec-Fetch-Storage-Access: active` header. (This token may be ignored if the previous request did not include `Sec-Fetch-Storage-Access: inactive`.)
```

Comment:
Why do we have this clause? It seems unnecessary to point out that if the site doesn't have permission, access isn't granted, but maybe I'm misunderstanding.

Reply:
Not sure which clause in particular you're referring to - but I just want to make it clear that this isn't a way to *request* permission, so if the permission isn't granted, there's nothing the server can do (other than run `document.requestStorageAccess()` on the client later). I did say that in the non-goals section of the explainer, but IMO it doesn't hurt to be explicit here too.

Reply:
I see. I think you should then say that explicitly rather than implying it through "if the previous request did not include `Sec-Fetch-Storage-Access: inactive`", if that makes sense? Maybe just add a clarifying sentence that it only works if permission is actually granted :)

Reply:
Done, rephrased this a bit so it's expressed more clearly (I hope).